src/models/AnalyticsEventType.ts

```diff
@@ -28,5 +28,8 @@ export enum AnalyticsEventType {
   Rsvp = 'RSVP',
 
   // Pagination event types
-  Paginate = 'PAGINATE'
+  Paginate = 'PAGINATE',
```

Comment:
nit: I think the CTA event types are listed separately because there are a lot of them, but since the other events just have one or two types, we've just grouped them together at the top, which seems a little more concise than separating by event. but I don't have a strong preference if you'd rather keep them separate

Reply:
sure